target/linux/mediatek/filogic/base-files/lib/upgrade/platform.sh

```diff
@@ -63,7 +64,7 @@ platform_do_upgrade() {
 		nand_do_upgrade "$1"
 		;;
 	bananapi,bpi-r3)
-		local rootdev="$(cmdline_get_var root)"
+		rootdev="$(cmdline_get_var root)"
```

Comment:
looks like a mistake

Reply:
We are not inside a function, so `local` doesn't make sense.
It should probably go into a separate commit.